### ISSUE TITLE
PHP parse_deposit_addresses usage bug.

### ISSUE THREAD

**Issue by jskorlol** [ccxt/ccxt#10721]
```php
    public function parse_deposit_addresses($addresses, $codes = null, $indexed = true, $params = array()){
        $result = array();
        for ($i = 0; $i < count($addresses); $i++) {
            $address = $this->extend($this->parse_deposit_address($addresses[$i]), $params);
            $result[] = $address;
        }
        if ($codes) {
            $result = $this->filter_by_array($result, 'currency', $codes, false);
        }
        return $indexed ? $this->index_by($result, 'currency') : $result;
    }
```

```php
    public function parse_deposit_addresses($addresses, $codes = null, $indexed = true, $params = array()){
        $result = array();
        for ($i = 0; $i < count($addresses); $i++) {
            $address = $this->deep_extend($this->parse_deposit_address($addresses[$i]), $params);
            $result[] = $address;
        }
        if ($codes) {
            $result = $this->filter_by_array($result, 'currency', $codes, false);
        }
        return $indexed ? $this->index_by($result, 'currency') : $result;
    }
```

$this->extend

$this->deep_extend

changed to

Exchange.php 2054 line.

upbit

`fetch_deposit_addresses`

If it works, an error occurs.

**Comment by kroitor** [comment]
@jskorlol please, paste your versions, your code to reproduce the bug and your verbose request + response:

- https://github.com/ccxt/ccxt/wiki/FAQ#what-is-required-to-get-help

**Comment by jskorlol** [comment]
```
   ccxt\ExchangeError

  extend method not found, try underscore_notation instead of camelCase for the method being called

  at vendor/ccxt/ccxt/php/Exchange.php:2599
    2595▕             $underscore = static::$camelcase_methods[$function];
    2596▕             return call_user_func_array(array($this, $underscore), $params);
    2597▕         } else {
    2598▕             /* handle errors */
  ➜ 2599▕             throw new ExchangeError($function . ' method not found, try underscore_notation instead of camelCase for the method being called');
    2600▕         }
    2601▕     }
    2602▕
    2603▕     public function __sleep() {
```

```php
dump($this->upbit->fetch_deposit_addresses());
```

version : 1.62.38

Supplement the content. I'm sorry.

**Comment by kroitor** [comment]
@jskorlol fixed in 1.62.39+, the new build will arrive in ~20 minutes. Please, let us know if that solves the issue for your or not. Thanks for the feedback!
